### PR TITLE
Fixup version info

### PR DESCRIPTION
**Pull request by KotlinIsland**
The based version was imposting on plugins.

### CODE REVIEW

**Comment by DetachHead** [comment]
what does `mypy --version` output look like now (in both dev and release)

**Comment by KotlinIsland** [comment]
The same, only difference are the 'internals' that plugins look at.

File: mypy/version.py [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+
 from mypy import git
 
 # Base version.
 # - Release versions have the form "0.NNN".
 # - Dev versions have the form "0.NNN+dev" (PLUS sign to conform to PEP 440).
 # - For 1.0 we'll switch back to 1.2.3 form.
-__version__ = '2.0.0+dev'
+__version__ = '0.940+dev'
 base_version = __version__
```
Comment (DetachHead) [inline mypy/version.py:10]: i assume `base_version` is not to be confused with `based_version`? perhaps one of them can be renamed?

Comment (KotlinIsland) [reply mypy/version.py:10]: `base_version` doesn't even exist at install time so idk.

**Pull request merged**